src/contexts/SavedGridContext.js

```diff
@@ -0,0 +1,29 @@
+import { createContext, useContext, useState } from 'react';
+import PropTypes from 'prop-types';
+
+const SavedGridContext = createContext();
+
+export function useSavedGrid() {
+  return useContext(SavedGridContext);
```

Comment:
Well done with creating a custom hook that calls directly the context! 🥳 This is a common pattern. When you feel, ready, you can dig deeper into [using React Context effectively](https://kentcdodds.com/blog/how-to-use-react-context-effectively)